Signal/src/ViewControllers/MessageMetadataViewController.swift

```diff
@@ -267,76 +276,79 @@ class MessageMetadataViewController: OWSViewController {
         }
     }
 
+    private func displayableTextIfText() -> String? {
+        let messageCellType = viewItem.messageCellType()
```

Comment:
We weren't doing the "strip, de-zalgo" thing here.  By using `ConversationViewItem`, we can lean on its cache and DRY up this logic.